kw/platform/requests/patch.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import requests
+import wrapt
+
+
+def construct_user_agent(app_name=None, package_version=None, app_environment=None):
+    """Helper function for constructing User-Agent header from args or env vars."""
+    app_name = app_name or os.getenv("APP_NAME")
+    package_version = package_version or os.getenv("PACKAGE_VERSION")
+    app_environment = app_environment or os.getenv("ENVIRONMENT")
```

Comment:
Oh, last time when I mentioned calling it 'app environment', I meant the environment var, too.